MockKit/MockCGMManager.swift

```diff
@@ -572,6 +572,10 @@ public final class MockCGMManager: TestingCGMManager {
         guard !pastSamples.isEmpty else { return }
         sendCGMReadingResult(CGMReadingResult.newData(pastSamples.map { NewGlucoseSample($0, device: device) } ))
     }
+    
+    public func acceptDefaultsAndSkipOnboarding() {
+        // TODO: Implement This
+    }
```

Comment:
is there something to implement here? The comment may be misleading.

Reply:
It may be useful to have implemented some day but in an attempt to get this study product developed on-time I thought it might be best to save it for another day. I can create a ticket to implement this and update the comment to reflect this future work is planned. Does that sound okay?